railties/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Deprecate `SOURCE_ANNOTATION_DIRECTORIES` environment variable used by `rails notes`
+    in favor of using `config.annotations.register_directories`
+
+    Deprecate `rake notes` in favor of `rails notes`
```

Comment:
Hmm, this reads partly weird because of our Rake proxy that allows apps to run rake tasks via `rails xxx`.

Reply:
Yeah I was thinking about that too but currently you can still do `rake notes`. After we enact the deprecation, you wouldn't anymore because we would get rid of the rake task itself.